TestBenches/TrackMerger_test.cpp

```diff
@@ -20,40 +20,43 @@ int main(){
     
   // Input memories
   TrackFitType::TrackWord trackWord[kMaxTrack];
-  TrackFitType::BarrelStubWord barrelStubWords[kMaxTrack][4];
-  TrackFitType::DiskStubWord diskStubWords[kMaxTrack][4];
+  TrackFitType::BarrelStubWord barrelStubWords[kMaxTrack][6];
+  TrackFitType::DiskStubWord diskStubWords[kMaxTrack][5];
   TrackFitMemory<NBarrelStub, NDiskStub> inputTracks;
 
   // Output memories
   TrackFitType::TrackWord trackWord_o[kMaxTrack];
-  TrackFitType::BarrelStubWord barrelStubWords_o[kMaxTrack][4];
-  TrackFitType::DiskStubWord diskStubWords_o[kMaxTrack][4];
+  TrackFitType::BarrelStubWord barrelStubWords_o[kMaxTrack][6];
+  TrackFitType::DiskStubWord diskStubWords_o[kMaxTrack][5];
   TrackFitMemory<NBarrelStub, NDiskStub> outputTracks;
   // int outputNumber;
  
   TBHelper tb("../../../../../emData/PD/PD/");
 
   // Open input files
-  auto &fin_inputTracks = tb.files("TrackFit_TF_L1L2*");
+  auto &fin_inputTracks = tb.files("TrackFit_TF_AAAA*");
 
 
-  auto &fout_outputTracks = tb.files("CleanTrack_CT_L1L2*"); 
+  auto &fout_outputTracks = tb.files("CleanTrack_CT_AAAA*"); 
   for (unsigned int ievt = 0; ievt < nevents; ++ievt) { 
     cout << "Event: " << dec << ievt << endl;
     
     for (unsigned short i = 0; i < kMaxTrack; ++i){
       trackWord[i] = TrackFitType::TrackWord(0);
       trackWord_o[i] = TrackFitType::TrackWord(0);
 
-      for (unsigned short nStub = 0; nStub < 4; nStub++){
+      for (unsigned short nStub = 0; nStub < 6; nStub++){
```

Comment:
@aryd Please use the constants from TrackletAlgorithm/Constants.h here:
```suggestion
      for (unsigned short nStub = 0; nStub < trklet::N_LAYER; nStub++){
```

Reply:
Fixed.